persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/multinode/AbstractClusteredPersistentEntitySpec.scala

```diff
@@ -143,13 +150,13 @@ abstract class AbstractClusteredPersistentEntitySpec(config: AbstractClusteredPe
 
   "A PersistentEntity in a Cluster" must {
 
-    "send commands to target entity" in within(21.seconds) {
+    "send commands to target entity" in within(75.seconds) {
```

Comment:
Set to 75 considering how many `ask` and `expect` operations we have on the test.